packages/core/ai_content_pipeline/ai_content_pipeline/motion_transfer.py

```
@@ -0,0 +1,340 @@
+    # Check dependencies
+    ok, error = check_dependencies()
+    if not ok:
+        return MotionTransferResult(success=False, error=error)
+
+    try:
+        # Upload files if local
+        image_url = upload_if_local(image_path, "image")
+        video_url = upload_if_local(video_path, "video")
+
+        # Call API
+        print(f"\n🎬 Starting motion transfer...")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)